src/randomCardsIds.js

```diff
@@ -0,0 +1,18 @@
+import { random, shuffle } from "lodash";
+
+const AMOUNT_OF_PHOTOS = 992;
+
+const randomCardsIds = length => {
+  let array = [];
+
+  while (array.length < length / 2) {
```

Comment:
There is probably a way to write this in a more functional way. Check out the lodash library for functions.